corehq/apps/es/tests/test_user_es.py

```diff
@@ -0,0 +1,52 @@
+import uuid
+
+from django.test import TestCase
+
+from pillowtop.es_utils import initialize_index_and_mapping
+
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.apps.es import UserES
+from corehq.apps.es.tests.utils import es_test
+from corehq.apps.users.dbaccessors.all_commcare_users import delete_all_users
+from corehq.apps.users.models import CommCareUser
+from corehq.elastic import get_es_new
+from corehq.pillows.mappings.user_mapping import USER_INDEX, USER_INDEX_INFO
+from corehq.util.elastic import ensure_index_deleted
+from corehq.util.es.testing import sync_users_to_es
+
+
+@es_test
+class TestUserES(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        initialize_index_and_mapping(get_es_new(), USER_INDEX_INFO)
+        cls.elasticsearch = get_es_new()
```

Comment:
This probably works fine, but I think it would be better to use the same instance of `get_es_new()` in these 2 lines

```suggestion
        cls.elasticsearch = get_es_new()
        initialize_index_and_mapping(cls.elasticsearch, USER_INDEX_INFO)
```